Formula/h/hg-fast-export.rb

```diff
@@ -6,7 +6,7 @@ class HgFastExport < Formula
   url "https://github.com/frej/fast-export/archive/refs/tags/v221024.tar.gz"
   sha256 "0dfecc3a2fd0833434d7ef57eb34c16432a8b2930df22a56ccf1a2bbb4626ba7"
   license "GPL-2.0-or-later"
-  revision 1
+  revision 2
```

Comment:
Incorrect migration in #151072 as some installed scripts have `from mercurial import ...` so needed to be migrated along with `mercurial` formula.

Reply:
May be a bit worse as `hg-fast-export` is not compatible with Python 3.12 due to usage of removed `imp` - https://github.com/frej/fast-export/blob/master/pluginloader/__init__.py#L2